main.go

```diff
@@ -399,4 +400,12 @@ var bitswapCheckCmd = &cli.Command{
 
 		return nil
 	},
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:        "get-block",
+			Usage:       "get the block",
+			Value:       true,
+			DefaultText: "true",
```

Comment:
Not sure if this should be true/false by default. Seems like this is mostly for debugging so increasing verbosity and depth of probing seems reasonable.